editors/vscode/package.json

```diff
@@ -3,7 +3,7 @@
 	"publisher": "biomejs",
 	"displayName": "Biome",
 	"description": "Biome LSP VS Code Extension",
-	"version": "1.0.1",
+	"version": "1.2.0",
```

Comment:
```suggestion
	"version": "1.1.0",
```

Reply:
This is intended: https://biomejs.dev/internals/versioning/#visual-studio-code-extension

Reply:
Oh yes forgot about the VSCode case ^^